### PR TITLE
ROSA-HCP portworx

### PR DESCRIPTION
**Pull request by nedoshi**
I have created documentation on installing portworx on ROSA_HCP cluster. Please review and provide feedback

### CODE REVIEW

File: content/rosa/rosa-hcp-portworx/index.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+tags: ["ROSA-HCP", "Portworx", "ROSA"]
+authors:
+  - Nerav Doshi
+---
+
+Portworx storage is a built-for-Kubernetes service that offers flexible and scalable persistent storage for applications in production. In this tutorial we will look at installing Portworx Enterprise on ROSA-HCP. 
+
+## Prerequisites
+You must have a Red Hat OpenShift Service on AWS (ROSA) with [hosted control plane cluster](https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-sts-overview-of-the-default-cluster-specifications_rosa-hcp-sts-creating-a-cluster-quickly) 
+Your cluster must be running OpenShift 4 or higher
```
Comment (paulczar) [inline content/rosa/rosa-hcp-portworx/index.md:13]: This is probably redundant as ROSA only supports 4.x

File: content/rosa/rosa-hcp-portworx/index.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+Your cluster must meet AWS prerequisites for ROSA-HCP
```
Comment (paulczar) [inline content/rosa/rosa-hcp-portworx/index.md:14]: also redundant, having a cluster likely means they meet the pre-reqs.

File: content/rosa/rosa-hcp-portworx/index.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+2. Click Get Started. Select Portworx Enterprise from the Product Catalog page.
+
+3. On the Product Line page, choose any option depending on which license you intend to use, then click Continue to start the spec generator.
+
+4. For Platform, choose **AWS**. Under Distribution Name, select **Red Hat OpenShift Service on AWS (ROSA)**. Enter your cluster's Kubernetes version, then click **Customize** at the bottom of the window.
+
+3. Click Next and navigate to the Customize window to specify your previously created AWS security credentials in the Environment Variables section as follows:
+
+    - name: AWS_ACCESS_KEY_ID; value: <your-aws-access-key>
```
Comment (paulczar) [inline content/rosa/rosa-hcp-portworx/index.md:31]: is there a better way to do this than past AWS creds into a portworx website ?

File: content/rosa/rosa-hcp-portworx/index.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+    - name: AWS_SECRET_ACCESS_KEY; value: <your-aws-secret-access-key>
+
+4. Click Finish to generate the specs and download yaml file.
+
+## Open ports for worker nodes
+Perform the following to add the inbound rules so that the AWS EC2 instance uses your specified security groups to control the incoming traffic.
+
+1. From the EC2 page of your AWS console, click Security Groups, under Network & Security, in the left pane.
+
+2. On the Security Groups page, type your ROSA cluster name in the search bar and press enter. You will see a list of security groups associated with your cluster. Click the link under Security group ID of your cluster's worker security group:
```
Comment (paulczar) [inline content/rosa/rosa-hcp-portworx/index.md:41]: don't we have a way to add additional security group rules via rosa cli now ?

File: content/rosa/rosa-hcp-portworx/index.md (hunk 1 above)
Comment (nedoshi) [reply content/rosa/rosa-hcp-portworx/index.md:13]: Thanks for the suggestion. I will remove the pre-req

File: content/rosa/rosa-hcp-portworx/index.md (hunk 2 above)
Comment (nedoshi) [reply content/rosa/rosa-hcp-portworx/index.md:14]: I will remove the pre-req

File: content/rosa/rosa-hcp-portworx/index.md (hunk 4 above)
Comment (nedoshi) [reply content/rosa/rosa-hcp-portworx/index.md:41]: I will verify this with rosa cli and make changes

File: content/rosa/rosa-hcp-portworx/index.md (hunk 3 above)
Comment (nedoshi) [reply content/rosa/rosa-hcp-portworx/index.md]: Incorporated adding secrets as environment variables

File: content/rosa/rosa-hcp-portworx/index.md (hunk 4 above)
Comment (nedoshi) [reply content/rosa/rosa-hcp-portworx/index.md]: Based on Portworx recommendation need to open ports using inbound rules for security group associated with worker nodes. added a section on making changes using aws cli

**Review by paulczar — APPROVED** [review]

**Pull request merged**